LEAF_Request_Portal/sources/FormWorkflow.php

```diff
@@ -65,7 +67,7 @@ public function isActive()
     /**
      * Retrieves current steps of a form's workflow, controls access to steps
      * @return array database result
-     * @return null if no database result
+     * @return array|int|null if no database result
      */
     public function getCurrentSteps()
```

Comment:
Return params

Reply:
Again, scope. I probably added it here because of VSCode not really liking just null in this case.